packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -135,10 +135,18 @@ export class AtomicSearchInterface {
   @Watch('searchHub')
   @Watch('pipeline')
   public updateSearchConfiguration() {
+    if (!this.engine) {
```

Comment:
Could we wrap `this.engine` in a getter that perform that check in that class ? Meaning we won't forget to add those checks in the future when accessing the engine.

And as for the warning, just a generic `You have to call "initialize" on the atomic-search-interface component before modifying any property` could do.

Reply:
Good idea!

Reply:
Hmm since engine is a Prop and is used for Atomic React I guess we can't break that interface by renaming "engine"
But I'll had a method